gravitee-am-ui/src/app/guards/license-guard.service.ts

```diff
@@ -14,13 +14,13 @@
  * limitations under the License.
  */
 import { Injectable } from '@angular/core';
-import { ActivatedRouteSnapshot, CanActivate, Route } from '@angular/router';
+import { ActivatedRouteSnapshot, Route } from '@angular/router';
 import { Observable, of } from 'rxjs';
 import { GioLicenseService, LicenseOptions } from '@gravitee/ui-particles-angular';
 import { map } from 'rxjs/operators';
 
 @Injectable()
-export class LicenseGuard implements CanActivate {
```

Comment:
CanActive should be replaces with CanActivateFn
https://stackoverflow.com/questions/75564717/angulars-canactivate-interface-is-deprecated-how-to-replace-it